components/eventing-controller/controllers/subscriptionv2/jetstream/reconciler_integration_test.go

```diff
@@ -316,38 +397,6 @@ func TestChangeSubscription(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "CleanEventTypes; add filters to subscription without filters",
```

Comment:
What is the reason for removing this test? Are they covered somewhere else?

Reply:
This test is no longer valid since we can never create a subscription without filters